reference/carts.v3.yml

```
@@ -2969,6 +3186,313 @@ components:
                   format: float
       title: Item Physical Get
       x-internal: false
+    NotFound:
+      description: Error payload for the BigCommerce API.
+      type: object
+      properties:
+        status:
+          description: |
+            404 HTTP status code.
+          type: integer
+        title:
+          description: The error title describing the particular error.
+          type: string
+        type:
+          type: string
+        instance:
+          type: string
+      title: Not Found
+      x-internal: false
+    MetafieldResponse:
+      description: |
+        Response payload for the BigCommerce API.
+      x-internal: false
+      x-examples: {}
+      allOf:
+        - type: object
+          properties:
+            data:
+              $ref: '#/components/schemas/Metafield'
+        - $ref: '#/components/schemas/Meta'
+    Metafield:
+      description: |
+        Allows app partners to write custom data to various resources in the API.
+      allOf:
+        - $ref: '#/components/schemas/MetafieldBase'
+        - type: object
+          properties:
+            id:
+              type: integer
+              description: |
+                The unique identifier for the metafields.
+            date_created:
+              type: string
+              format: date-time
+              description: |
+                Date and time of the metafield's creation.
+              example: '2022-06-16T18:39:00+00:00'
+            date_modified:
+              type: string
+              format: date-time
+              description: |
+                Date and time when the metafield was last updated.
+              example: '2022-06-16T18:39:00+00:00'
+      x-internal: false
+      x-examples: {}
+    MetafieldBase:
+      type: object
+      description: |
+        Common Metafield properties.
+      x-internal: false
+      properties:
+        permission_set:
+          type: string
+          description: |
+            Determines the visibility and writeability of the field by other API consumers.
+
+            |Value|Description
+            |-|-|
```

```suggestion
            | :--- | :--- |
```
I believe we've been doing this for our tables